### ISSUE TITLE
Report 0x11

### ISSUE THREAD

**Issue by gusmanb** [gusmanb/PSVRFramework#29]
So, this report does a lot more than we thought.

Byte 1 and 3 control the led settings. Byte 1 is a two nibble value, first nibble turns on/off the front leds, second nibble rear leds. Byte 2 is a two nibble mask, nibble 1 indicates if to apply the byte 1 front led setting and nibble 2 indicates if to apply the byte 1 back led settings.

Now, byte 0 and 2 have me puzzled, byte 2 seems to be a mask, when bit 1 is set byte 0 starts to work, and byte 0 seems to do something with screen mode, whenever it's value changes the screen resets but the HMD doesn't show again the screen after that reset, it keeps the loader on screen.

The other four bytes left seem to do nothing.

**Comment by mungewell** [comment]
Also note that is changes status report 0xf0.

Which screen, HMD or Social Screen?

**Comment by gusmanb** [comment]
Resets both, but the HMD screen keeps on loading

**Comment by mungewell** [comment]
Did you try sending 90Hz or 120Hz display?

Might also be interesting to see if EDID changes.

**Comment by gusmanb** [comment]
Nope, didn't tested, but I found I had an stupid error on the code and the final bytes indeed do something, I have now them set to 0xFF and b2 also to 0xFF and changing b0 bits 0,1 and 2 change the screen mode, 0 changes to VR, 1 to cinematic and 2 again to VR.

**Comment by mungewell** [comment]
Not quiet following the description, can you write up in a wiki page?

Do I remember something about a 120Hz low persistance mode, or was the other VR headsets.

**Comment by gusmanb** [comment]
I will write Monday as I plan to test this more and wnat to have something a bit more clear before adding it to the wiki.

About 120Hz, no idea, I think that may be from other headset.
